sql/item_mods.sql

```diff
@@ -19998,16 +19998,40 @@ INSERT INTO `item_mods` VALUES (16397,25,3);
 INSERT INTO `item_mods` VALUES (16398,431,1);
 INSERT INTO `item_mods` VALUES (16403,431,1);
+INSERT INTO `item_mods` VALUES (16404,431,1);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,499,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,501,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,951,3);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,952,4);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,953,30);  -- Venom Katars - Additional effect: Poison
 INSERT INTO `item_mods` VALUES (16410,431,1);
 INSERT INTO `item_mods` VALUES (16416,54,5);
+INSERT INTO `item_mods` VALUES (16416,431,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,499,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,500,30);  -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,501,10);  -- Dragon Claws - Additional effect: Fire damage
 INSERT INTO `item_mods` VALUES (16417,431,1);
+INSERT INTO `item_mods` VALUES (16418,431,1);   -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,499,10);  -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,501,10);  -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,951,3);   -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,952,4);   -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,953,30);  -- Venom Claws - Additional effect: Poison
 INSERT INTO `item_mods` VALUES (16423,10,2);
+INSERT INTO `item_mods` VALUES (16425,431,1);   -- Venom Claws +1 - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16425,499,10);  -- Venom Claws +1 - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16425,501,10);  -- Venom Claws +1 - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16425,951,3);   -- Venom Claws +1 - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16425,952,4);   -- Venom Claws +1 - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16425,953,30);  -- Venom Claws +1 - Additional effect: Poison
 INSERT INTO `item_mods` VALUES (16426,1,-10);
 INSERT INTO `item_mods` VALUES (16426,23,10);
 INSERT INTO `item_mods` VALUES (16428,165,3);
 INSERT INTO `item_mods` VALUES (16428,292,7);
 INSERT INTO `item_mods` VALUES (16429,431,1);
-INSERT INTO `item_mods` VALUES (16431,431,1); -- Stun claws - Additional effect: "Stun"
+INSERT INTO `item_mods` VALUES (16430,431,1);   -- Acid Claws - Additional effect: Weakens defense
+INSERT INTO `item_mods` VALUES (16431,431,1);   -- Stun claws - Additional effect: "Stun"
+INSERT INTO `item_mods` VALUES (16432,431,1);   -- Stun Jamadhars - Additional effect: "Stun"
```

Comment:
these 3 are missing the other mods for status effects. the weaken and the stun both needs same kind of info you gave poison. I.E. "How potent is the effect of def down? How long does it last?"